src/components/shared/Navigation.tsx

```diff
@@ -1,16 +1,251 @@
+  const isLoggedIn = true;
   const isChapter = false;
-  const isNonprofit = false;
+  const isNonprofit = true;
+  const npUserName = "liv.2b.girl";
+  const nonProftName = "Liv2BGirl";
+  const chapterName = "Chapter Name";
+
+  //colors
+  const black = "rgba(51, 51, 51, 1)";
+  const slate = "rgba(101, 119, 136, 1)";
+  const blue = "rgba(0, 105, 202, 1)";
 
   return (
-    <Flex height="70px" borderBottom="solid">
-      <Text>Navigation</Text>
-    </Flex>
+    <Box>
+      <Flex
+        bg={useColorModeValue("white", "gray.800")}
+        color={useColorModeValue("gray.600", "white")}
+        maxH={"66px"}
+        py="23px"
+        px="34px"
+        borderBottom={1}
+        borderStyle={"solid"}
+        borderColor={useColorModeValue("gray.200", "gray.900")}
+        align={"center"}
+      >
+        <Flex flex={{ base: 1 }} justify={{ base: "center", md: "start" }}>
+          <Image src={Logo} width={150} height={150} />
+          {isLoggedIn && (isChapter || isNonprofit) ? (
+            <Flex display={{ base: "none", md: "flex" }} ml={6}>
+              <DesktopNav
+                isLoggedIn={isLoggedIn}
+                isChapter={isChapter}
+                isNonprofit={isNonprofit}
+              />
+            </Flex>
+          ) : null}
+        </Flex>
+        {isLoggedIn ? (
+          <Flex alignItems={"center"}>
+            <Menu>
+              <VStack spacing={-2} align="flex-end">
+                <Text p={1} fontSize="sm" fontWeight={700} color={black}>
+                  {isChapter ? chapterName : nonProftName}
+                </Text>
+                <Text p={1} fontSize="sm" fontWeight={500} color={slate}>
+                  {isChapter ? "Admin" : npUserName}
+                </Text>
+              </VStack>
+              <MenuButton
+                as={Button}
+                rounded={"full"}
+                variant={"link"}
+                cursor={"pointer"}
+              >
+                <Avatar
+                  size={"sm"}
+                  src={"https://hack4impact.org/brokenLink"}
+                />
+              </MenuButton>
+              <MenuList>
+                <MenuItem>Edit Profile</MenuItem>
+                <MenuItem>Sign Out</MenuItem>
+              </MenuList>
+            </Menu>
+          </Flex>
+        ) : (
+          <Stack
+            flex={{ base: 1, md: 0 }}
+            justify={"flex-end"}
+            direction={"row"}
+            spacing={6}
+          >
+            <Button
+              display={{ base: "none", md: "inline-flex" }}
+              fontSize={"sm"}
+              fontWeight={600}
+              color={"white"}
+              bg={blue}
+              href={"#"}
+              _hover={{
+                bg: "rgba(0, 105, 202, 0.5)",
+              }}
+            >
+              Log In
+            </Button>
+          </Stack>
+        )}
+      </Flex>
+    </Box>
   );
 }
 
 export default Navigation;
+
+const DesktopNav = (props: {
```

Comment:
So there are two possible options for dealing with this separate component.  The first is just moving the jsx into the main component since there is no need for a separate component.  The second is creating a navigation folder and creating a separate file for this component and importing it.  I would do it the first way, but either is fine.  It is generally good practice to maintain the rule of one component per file if possible.